src/report.ts

```diff
@@ -0,0 +1,59 @@
+// (C) 2007-2018 GoodData Corporation
+import { XhrModule, ApiResponse } from './xhr';
+import { IExportConfig, IExportPromise, IExportResponse } from './interfaces';
+import { handlePolling, IPollingOptions } from './util';
+
+interface IResultExport {
```

Comment:
I would move the interfaces to interfaces file next to others as `IExportConfig` :) which is already there... what do you think?

Reply:
IResultExport and IExportResultPayload are used internally in report.ts.
If I move them to interfaces file, they will be exported outside too

// gooddata-browser.ts
export * from './interfaces';

In my opinion, I prefer to leave them there

Reply:
ok :)